### PR TITLE
{bio}[foss/2019a] gradunwarp 1.1.0 (HCP fork)

### PR DESCRIPTION
**Pull request by crubb**
As discussed on Slack. I added HCP as a Version suffix.

**Fork:** https://github.com/Washington-University/gradunwarp
**Original:** https://github.com/ksubramz/gradunwarp (no longer actively maintained: https://github.com/ksubramz/gradunwarp/commit/6bb9c35f6113980058a6099b7a3593027be288fa)

### CODE REVIEW

File: easybuild/easyconfigs/g/gradunwarp/gradunwarp-1.1.0-foss-2019a-HCP-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+description = """Gradient Unwarping. This is the Human Connectome Project fork of the no longer maintained original."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://github.com/Washington-University/gradunwarp/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4803b8055dbeedb0435246a525aa69f1f3425c55d57c973c045deb39bc95c955']
+
+download_dep_fail = True
+use_pip = False
```
Comment (smoors) [inline easybuild/easyconfigs/g/gradunwarp/gradunwarp-1.1.0-foss-2019a-HCP-Python-2.7.15.eb]: @crubb please add `sanity_pip_check = True` here?

Comment (crubb) [reply easybuild/easyconfigs/g/gradunwarp/gradunwarp-1.1.0-foss-2019a-HCP-Python-2.7.15.eb]: Hey, we're not using pip to install, therefore I suppose `sanity_pip_check` should be the default, i.e. `False`? 

See https://github.com/easybuilders/easybuild-easyconfigs/issues/9563

Comment (smoors) [reply easybuild/easyconfigs/g/gradunwarp/gradunwarp-1.1.0-foss-2019a-HCP-Python-2.7.15.eb]: ah, you're right, I missed that use_pip = False

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node355.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/60893fdd072195f87cd51c143606d365 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @crubb!

File: easybuild/easyconfigs/g/gradunwarp/gradunwarp-1.1.0-foss-2019a-HCP-Python-2.7.15.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/gradunwarp/gradunwarp-1.1.0-foss-2019a-HCP-Python-2.7.15.eb]: @crubb Can you clarify why you're using `use_pip = False` here, is there a specific reason for it?

Comment (crubb) [reply easybuild/easyconfigs/g/gradunwarp/gradunwarp-1.1.0-foss-2019a-HCP-Python-2.7.15.eb]: As discussed on Slack: https://github.com/easybuilders/easybuild-easyconfigs/pull/9637

**Pull request merged**